java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -236,6 +237,22 @@ public T scard(String key) {
         return getThis();
     }
 
+    /**
+     * Returns the number of keys in <code>keys</code> that exist in the database.
+     *
+     * @see <a href="https://redis.io/commands/exists/">redis.io</a> for details.
+     * @param keys The keys list to check.
+     * @return Command Response - the number of keys that exist. If the same existing key is mentioned
```

Comment:
- The (i think we had decided it would be capital after Command Response)

Reply:
regardless, it looks better capitalized